conf/battle/feature.conf

```diff
@@ -71,6 +71,18 @@ feature.achievement: on
 // Requires: 2017-02-08bRagexeRE or later
 feature.equipswitch: on
 
+// Pet evolution (Note 1)
+// Requires: 2014-10-08aRagexe or later
+feature.petevolution: on
+
+// Automatic Pet Feeding (Note 1)
+// Requires: 2014-10-08aRagexe or later
+feature.petautofeed: on
+
+// At which rate should pet autofeeding trigger? (Note 2)
+// Default: 89
+feature.pet_autofeed_rate: 89
```

Comment:
It should say `At which hunger should pet autofeeding trigger? (Note 2)`, even if the hunger is the rate